### PR TITLE
my code is basically perfect

### PR DESCRIPTION
**Pull request by walkerkit**
Did a quick review of the mutations, I'm happy

### CODE REVIEW

**Comment by walkerkit** [comment]
The __eq__ and subsequent __hash__ functions haven proven too jank... the situation has become untenable! 
Society has nearly broken down, if not for this PR.

For more information see this vaguely relevant SO question: [here!](https://stackoverflow.com/questions/61212514/django-model-objects-became-not-hashable-after-upgrading-to-django-2-2)

File: uobtheatre/utils/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        ticketKey = (
+            ticket.get("seat_group_id"),
+            ticket.get("concession_type_id"),
+            ticket.get("seat_id"),
+        )
+        if ticketKey in ticketDict:
+            ticketDict[ticketKey] += 1
+        else:
+            ticketDict[ticketKey] = 1
+    return ticketDict
```
Comment (JElgar) [inline uobtheatre/utils/test_utils.py:13]: Could we do something like this?

```suggestion
    ticketObjectList = map(Ticket, ticketDictList)
    return ticketListDictGen(ticketObjectList)
```

**Review by JElgar — CHANGES_REQUESTED** [review]
Very nice thank you Kit. Only one thing is it looks like those functions duplicate logic a bit. What do you think about something like the suggestion?

File: uobtheatre/utils/test_utils.py (hunk 1 above)
Comment (JElgar) [reply uobtheatre/utils/test_utils.py:13]: You might actually need to use list comp here eg:
`ticketObjectList = [Ticket(**ticketDict) for ticketDict in ticketDictList]`

Comment (walkerkit) [reply uobtheatre/utils/test_utils.py]: Lovely suggestion there James!

**Review by JElgar — APPROVED** [review]

**Pull request merged**